packages/org-unit-tree/src/utils.js

```diff
@@ -62,7 +82,12 @@ export const mergeChildren = (root, children) => {
  * @param forceReloadChildren
  * @returns {*}
  */
-export const loadChildren = (root, displayNameProperty, forceReloadChildren) => {
+export const loadChildren = (
+    root,
+    displayNameProperty,
+    forceReloadChildren,
+    useUserDataViewFallback
```

Comment:
Maybe call it "isUserDataViewFallback" so it doesn't look like a custom react hook.

Reply:
ok

Reply:
Changed